spec/api-browser-window-spec.ts

```diff
@@ -1557,6 +1567,17 @@ describe('BrowserWindow module', () => {
         expectBoundsEqual(w.getSize(), size);
       });
 
+      it('emits the resize event for single-pixel size changes', async () => {
+        const [width, height] = w.getSize();
+        const size = [width + 1, height + 1];
```

Comment:
What do you think about testing `- 1`, too?

```suggestion
        const size = [width + 1, height - 1];
```